client/main.py

```diff
@@ -13,21 +14,18 @@
 game_win = pg.display.set_mode(res)
 
 with zmq.Context() as zmq_ctx:
-
-    address = loginPage(game_win)
+    rejoin = False
     should_stop = not SHOULD_STOP_GAME
-
     while not should_stop:
-
+        if not rejoin:
+            address = loginPage(game_win)
```

Comment:
note here that we expect an `address` from loginPage. If you exit the game from the loginpage the game crashes because a boolean value is returned.